components/lib/dataprotect/src/main/java/mozilla/components/lib/dataprotect/Keystore.kt

```diff
@@ -261,6 +263,7 @@ open class Keystore(
      * @throws GeneralSecurityException If the Cipher could not be created and initialized
      */
     @Throws(GeneralSecurityException::class)
+    @Synchronized
```

Comment:
If all of the users of what's returned out of these two methods are synchronized - and they in this PR - i don't think you need to synchronize here (or in createDecryptCipher).